radis/misc/profiler.py

```diff
@@ -48,7 +48,12 @@ def __init__(self, verbose):
     # Creates profiler dictionary structure
     def add_entry(self, dictionary, key, verbose, count):
         if count == verbose:
-            dictionary[key] = {}
+            try:
+                if dictionary[key]:
+                    return
+            except:
+                dictionary[key] = {}
```

Comment:
can you have an "if" only verison ; or define the error you expect ? Also, there is no "else" here